docs/GettingStarted.md

```diff
@@ -93,6 +93,14 @@ npm install -g react-native-cli
 
 If you get a permission error, try with sudo: `sudo npm install -g react-native-cli`.
 
+If you want to use [Flow](https://flowtype.org/):
+
+```
+npm install --global flow-bin
```

Comment:
You might consider to use local flow and run it from `node_modules/.bin/`. This allows to use different flow versions on multiple React Native projects